src/metabase/util.clj

```diff
@@ -498,6 +498,12 @@
   "A regular expression for matching canonical string representations of UUIDs."
   #"[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}")
 
+(defn ensure-seq
```

Comment:
We already have a util function that does this, `u/one-or-many`

https://github.com/metabase/metabase/blob/master/src/metabase/util.clj#L646

I'm not sure that's the greatest name so we should pick one and remove the other

Reply:
How do you feel about changing one-or-many, to consider nil a seq, I think it's what you usually want for sane null punning and such

Reply:
Yeah that makes sense 👍